arguments/SPIR-V.argdown

```diff
@@ -56,8 +60,11 @@ functionality present in core SPIR-V, like the instructions for ray tracing.
 (3) That will effectively lead to the forking SPIR-V
   -> [SPIR-V]
 
-<Nobody wants yet another shading language>: we already have a diversity of shading languages,
```

Comment:
you replaced one point by an entirely different one, that's unexpected
"Nobody wants yet another shading language" has nothing to do with gatekeeping